iac-common/src/main/java/org/sonar/iac/common/extension/visitors/MetricsVisitor.java

```diff
@@ -46,7 +48,17 @@ protected MetricsVisitor(FileLinesContextFactory fileLinesContextFactory, NoSona
     languageSpecificMetrics();
   }
 
-  protected abstract void languageSpecificMetrics();
+  protected void languageSpecificMetrics() {
+    register(IacToken.class, (ctx, token) -> {
+      if (!(token.value().trim().isEmpty())) {
```

Comment:
I agree, StringUtils would look smarter. However, we try to avoid using apache commons-lang. So far our parsers should not create and store in the AST a token with whitespace or even empty. However, this common class should catch the cases if one of the used parsers is creating such a token. I don't get the clue with the [Whitespace](https://en.wikipedia.org/wiki/Whitespace_(programming_language)).